src/com/sun/ts/tests/signaturetest/signature-repository/jakarta.enterprise.concurrent.sig_2.0_se8

```diff
@@ -0,0 +1,167 @@
+hfds serialVersionUID
+
+CLSS public abstract interface jakarta.enterprise.concurrent.ContextService
+meth public abstract !varargs java.lang.Object createContextualProxy(java.lang.Object,java.lang.Class<?>[])
+meth public abstract !varargs java.lang.Object createContextualProxy(java.lang.Object,java.util.Map<java.lang.String,java.lang.String>,java.lang.Class<?>[])
+meth public abstract <%0 extends java.lang.Object> {%%0} createContextualProxy({%%0},java.lang.Class<{%%0}>)
+meth public abstract <%0 extends java.lang.Object> {%%0} createContextualProxy({%%0},java.util.Map<java.lang.String,java.lang.String>,java.lang.Class<{%%0}>)
+meth public abstract java.util.Map<java.lang.String,java.lang.String> getExecutionProperties(java.lang.Object)
+
+CLSS public abstract interface jakarta.enterprise.concurrent.LastExecution
+meth public abstract java.lang.Object getResult()
+meth public abstract java.lang.String getIdentityName()
+meth public abstract java.util.Date getRunEnd()
+meth public abstract java.util.Date getRunStart()
+meth public abstract java.util.Date getScheduledStart()
+
+CLSS public abstract interface jakarta.enterprise.concurrent.ManageableThread
+meth public abstract boolean isShutdown()
+
+CLSS public abstract interface jakarta.enterprise.concurrent.ManagedExecutorService
+intf java.util.concurrent.ExecutorService
+
+CLSS public jakarta.enterprise.concurrent.ManagedExecutors
+meth public static <%0 extends java.lang.Object> java.util.concurrent.Callable<{%%0}> managedTask(java.util.concurrent.Callable<{%%0}>,jakarta.enterprise.concurrent.ManagedTaskListener)
+meth public static <%0 extends java.lang.Object> java.util.concurrent.Callable<{%%0}> managedTask(java.util.concurrent.Callable<{%%0}>,java.util.Map<java.lang.String,java.lang.String>,jakarta.enterprise.concurrent.ManagedTaskListener)
+meth public static boolean isCurrentThreadShutdown()
+meth public static java.lang.Runnable managedTask(java.lang.Runnable,jakarta.enterprise.concurrent.ManagedTaskListener)
+meth public static java.lang.Runnable managedTask(java.lang.Runnable,java.util.Map<java.lang.String,java.lang.String>,jakarta.enterprise.concurrent.ManagedTaskListener)
+supr java.lang.Object
+hfds NULL_TASK_ERROR_MSG
+hcls Adapter,CallableAdapter,RunnableAdapter
+
+CLSS public abstract interface jakarta.enterprise.concurrent.ManagedScheduledExecutorService
+intf jakarta.enterprise.concurrent.ManagedExecutorService
+intf java.util.concurrent.ScheduledExecutorService
+meth public abstract <%0 extends java.lang.Object> java.util.concurrent.ScheduledFuture<{%%0}> schedule(java.util.concurrent.Callable<{%%0}>,jakarta.enterprise.concurrent.Trigger)
+meth public abstract java.util.concurrent.ScheduledFuture<?> schedule(java.lang.Runnable,jakarta.enterprise.concurrent.Trigger)
+
+CLSS public abstract interface jakarta.enterprise.concurrent.ManagedTask
+fld public final static java.lang.String IDENTITY_NAME = "javax.enterprise.concurrent.IDENTITY_NAME"
```

Comment:
This is wrong and the test fails due to the javax*. It was because the https://repo1.maven.org/maven2/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-api/2.0.0-RC1/jakarta.enterprise.concurrent-api-2.0.0-RC1.jar
 was used to generate the sig file. Latest RC2 release has this fixed but was not available in mvn yet. Will correct this.